### ISSUE TITLE
Stacktrace when running jfrog rt go-publish go . 

### ISSUE THREAD

**Issue by erikthorselius** [jfrog/jfrog-cli-go#250]
Somehow I get a panic because of the server header split. After some digging it turns out that our artifactory instance was behind a proxy and the server header was overwritten. But anyway this could also have been solved by using the version from the http body. The body is just thrown away to day. 

https://github.com/jfrog/jfrog-cli-go/blob/ba45bd3759f592470cb161894dcc77059ced967f/jfrog-client/artifactory/auth/rtdetails.go#L130

```
# jfrog rt go-publish go --self=false --deps=ALL
[Info] Using go: go version go1.11 linux/amd64

panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x86cda40, 0x18d36028)
	/var/lib/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/go-1.7.1/src/runtime/panic.go:500 +0x331
github.com/jfrog/jfrog-cli-go/jfrog-client/artifactory/auth.(*artifactoryDetails).getArtifactoryVersion(0x18e8a720, 0x0, 0x0, 0x0, 0x0)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/jfrog-client/artifactory/auth/rtdetails.go:130 +0x4f2
github.com/jfrog/jfrog-cli-go/jfrog-client/artifactory/auth.(*artifactoryDetails).GetVersion(0x18e8a720, 0x0, 0x0, 0x0, 0x0)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/jfrog-client/artifactory/auth/rtdetails.go:106 +0x4f
github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/commands/golang.Publish(0x18ddd200, 0x18e69765, 0x3, 0xfff5af9e, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/commands/golang/go.go:27 +0x103
github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory.goPublishCmd(0x18ddd220)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/cli.go:1300 +0x44b
github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory.GetCommands.func30(0x18ddd220)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/jfrog-cli/artifactory/cli.go:416 +0x21
github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli.Command.Run(0x874e150, 0xa, 0x0, 0x0, 0x18d98fd8, 0x1, 0x1, 0x876bf6f, 0x39, 0x8771862, ...)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli/command.go:174 +0xffb
github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli.(*App).RunAsSubcommand(0x18ddd0e0, 0x18ddd040, 0x0, 0x0)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli/app.go:298 +0xe4e
github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli.Command.startApp(0x874755c, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x8755249, 0x14, 0x0, ...)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli/command.go:249 +0x56a
github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli.Command.Run(0x874755c, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x8755249, 0x14, 0x0, ...)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli/command.go:65 +0x71
github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli.(*App).Run(0x18ddc780, 0x18d0a2d0, 0x6, 0x6, 0x0, 0x0)
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/vendor/github.com/codegangsta/cli/app.go:187 +0xd98
main.main()
	/var/lib/jenkins/workspace/EcoSystem/jfrog-cli-release/temp/src/github.com/jfrog/jfrog-cli-go/jfrog-cli/jfrog/main.go:81 +0x23a
# jfrog --version
jfrog version 1.20.2
```

**Comment by AlexeiVainshtein** [comment]
Thanks @erikthorselius for reporting this. We will change this in the upcoming releases to get the information from the body.

**Comment by AlexeiVainshtein** [comment]
Hi @erikthorselius I wanted to update you that [this commit](https://github.com/jfrog/jfrog-client-go/commit/3b3e007296a5f8eede07ed7fbf746464668a0744) resolved this issue.
This was already released with the latest CLI (version 1.23.2).
We will appreciate your feedback.

**Comment by erikthorselius** [comment]
Thanks for the follow up!
